app/assets/stylesheets/apply/apply.scss

```diff
@@ -348,7 +352,7 @@ label.remember-me {
 }
 
 .applications-closed {
-  color: black;
+  color: $black;
```

Comment:
Properties should be ordered background, border-color, border-radius, border-style, border-width, color, font-size, height, padding-bottom, padding-left, padding-right, padding-top, width